### PR TITLE
LEARNER-1940 - Restructure i18n files to ensure they're available in Django app

### PR DESCRIPTION
**Pull request by amangano-edx**
This PR restructures the i18n/locale files in credentials-themes to follow Django convention so that they may be used by the credentials service.

According to https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#how-django-discovers-translations, Django discovers translation files (.po/.mo files) by first looking in the directories configured via the LOCALE_PATHS setting, then by looking for a 'locale' directory in each installed application. The edx_credentials_themes directory is an installed application for the credentials service, therefore Django will expect any translations for the themes to live in 'edx_credentials_themes/locale'. 

JIRA: https://openedx.atlassian.net/browse/LEARNER-1940

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@ build.watch:
...
-	django-admin.py compilemessages
+	cd edx_credentials_themes && django-admin.py compilemessages
 
 extract_translations:
-	django-admin.py makemessages -l en -d django
+	cd edx_credentials_themes && django-admin.py makemessages -l en -d django
 
 i18n_requirements:
-	pip install -r ./conf/locale/requirements.txt
+	pip install -r ./i18n/requirements.txt
```
Comment (clintonb) [inline Makefile]: Move this file to requirements/i18n.txt.

File: edx_credentials_themes/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.1.4'
+__version__ = '0.1.6'
```
Comment (clintonb) [inline edx_credentials_themes/__init__.py]: 0.1.5

Comment (amangano-edx) [reply edx_credentials_themes/__init__.py]: 0.1.5 seems to already exist (there's a tag for it). I think someone may have just forgotten to bump the version.

**Comment by amangano-edx** [comment]
@clintonb I've addressed your feedback, and also added an entry to the MANIFEST.in file to ensure that the .mo files are included in the package when it's installed. When you have a second, could you please take one more look?

Also, post merge, do we typically create tags for each version bump?

**Comment by BenjiLee** [comment]
@amangano-edx I believe you also need to create a new release on this github repo after this merges.

**Pull request merged**